src/main/java/mate/academy/Main.java

```diff
@@ -6,18 +6,24 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
```

Comment:
Ensure that the logger is configured with both File and Console appenders as required by the task description.